packages/api/wrangler.toml

```diff
@@ -11,6 +11,10 @@ bindings = [
     { name = "IPNS_RECORD", class_name = "IPNSRecord" }
 ]
 
+[env.test.vars]
+REBROADCAST_INTERVAL_MS = 500
```

Comment:
Question here: should we perhaps add a jitter here?
Is there a scenario in where many alarms could trigger at the exact same time generating a huge amount of requests, just wondering if we could hit a bit of a thundering herd problem. 
IE if 1000s of records are added at once every 12 hours all requests may fire off at the same time causing spikes of requests, perhaps we could do it every 10 - 12 hours instead?

This could maybe something we monitor and add at a later date however.

Reply:
We will set REBROADCAST_INTERVAL_MS to be 24 hours on production and staging.

Here the setting is for the testing/local environment (`[env.test.vars]`). I set REBROADCAST_INTERVAL_MS to a small interval so we could test alarm. It might cause tests to hammer a bit your computer 😄

Reply:
Aye great point! 
I guess I was concerned around the potential issues on production, but actually the /broadcast endpoint now uses a queue so it shouldn't be too much of a concern.